datapipelines/src/test/java/io/customer/datapipelines/core/UnitTest.kt

```diff
@@ -52,14 +65,15 @@ abstract class UnitTest : BaseUnitTest() {
         val builder = CustomerIOBuilder(mock(), cdpApiKey)
         // Disable adding destination to analytics instance so events are not sent to the server by default
         builder.setAutoAddCustomerIODestination(false)
+        // Disable tracking application lifecycle events by default to avoid tracking unnecessary events while testing
+        builder.setTrackApplicationLifecycleEvents(false)
```

Comment:
the only pitfall is if we were to expect things from a default customer IO object, that would also be updated.

Reply:
Because the tests verifying this isn't using this as base class, that's not an issue at the moment. But I'll keep eyes on it on how can we improve this.